packages/tecrock-simulation/webpack.config.js

```diff
@@ -176,7 +176,11 @@ module.exports = {
       template: './src/index.html',
       favicon: './public/favicon.ico',
       publicPath: DEPLOY_PATH,
-    })] : [])
+    })] : []),
+    // Define DEPLOY_PATH globally so it can be used to load Web Worker script from a correct location
+    new webpack.DefinePlugin({
+      '__WEBPACK_DEPLOY_PATH__': JSON.stringify(DEPLOY_PATH || ".")
+    })
```

Comment:
@pjanik rather than doing this, perhaps you can use the built in webpack variable `__webpack_public_path__`? This is described here: https://github.com/concord-consortium/starter-projects/blob/master/doc/deploy.md

Reply:
Thanks @scytacki, I've made this change in https://github.com/concord-consortium/tectonic-explorer/pull/355.